### DISCUSSION TITLE
Massive Discrepancies in Visitor Numbers

### DISCUSSION THREAD

**Discussion by tnonate** [plausible/analytics#3951] [Self-Hosted Support]
I recently moved from a Namecheap Shared Hosting plan (where analytics were included against my will in CPanel) to a VPS with Plausible self-hosted on a separate VPS. I expected to see some variation - probably a dip - in numbers but I've seen over a 90% dropoff, which is far beyond what I expected. Specifically I've gone from somewhere in the neighborhood of 1300-1500 visitors per day to 130ish on a good day. I don't believe this to be bot-related because Namecheap claimed they counted bots separately and even had a little section for that specifically. Is there any chance I've set up my instance wrong? If it helps, my instance is https://stats.thenewoil.org/thenewoil.org and I already looked at [this](https://github.com/plausible/analytics/discussions/1722) issue which is similar but seems to deal with hosting multiple analytics services at once, which is not quite what I'm trying to do. If anyone can help, I'd super appreciate it. Happy to share whatever logs or answer any questions needed.

**Comment by ruslandoga** [comment]
👋 @thonate

You can try checking access logs or similar on your webserver and comparing them to the numbers you see on Plausible for some predefined interval.

**Comment by tnonate** [comment]
Reporting back: the issue was - in part - due to my domain name. Many adblockers blog "stats.*". Once I removed that, my numbers rose but still far short of the usual. I'm going to have to switch to something server-side. Just for anyone else who has a similar issue.
